src/main/java/org/apache/commons/collections4/bidimap/AbstractDualBidiMap.java

```diff
@@ -497,7 +494,7 @@ public boolean remove(final Object key) {
         protected boolean canRemove = false;
 
         /**
-         * Constructor.
+         * Constructor a KeySetIterator of the BidiMap
```

Comment:
"Constructor of a" or "Constructs a" would sound better I think.

However, I think just "Constructor" is much simpler/easier to maintain (ditto for the other similar changes in this PR)

Reply:
"Constructor." or "Constructs a new instance." is a good place holder comment, otherwise, you can describe what kind of initialization takes place.